incubator/java-spring-boot2/image/project/Dockerfile

```diff
@@ -3,17 +3,26 @@ FROM {{.stack.image.namespace}}/{{.stack.id}}:{{.stack.semver.major}}.{{.stack.s
 #setup project folder for java build step
 COPY . /project
 
+USER root
+RUN chown -R java_user:java_group /project
```

Comment:
If we're using the `chown` flag, please could you add a `docker-version` constraint on the `requirements` section in the `stack.yaml`. Should be a value of ">= 17.09.0"

Reply:
Aye, we were discussing this in the appsody slack (s'why this is just a WIP). 
We were wondering if it's better to find a different way to handle the chown, to retain compatibility with earlier Docker versions.
I need to investigate the impact to layering for 
 - using chown flag on each copy step
 - using individual run chown steps after each copy step
 - using a compound run chown step after all copy steps
The concern is the run chown steps will affect every file from the copy, which could negate the effect of layering the image (smaller uploads when only code is changed, not deps/libs etc).